### PR TITLE
Add `Tiles` component back

### PR DESCRIPTION
**Pull request by wkozyra95**
Tests: https://github.com/membraneframework-labs/video_compositor_snapshot_tests/pull/14

Changes:
- aspect ratio as a string e.g. `16:9`
- optional width/height (no support for top/left/right/bottom)
- change distance units to f32 where possible

### CODE REVIEW

File: compositor_render/src/scene/view_component/layout.rs [hunk 1]
```diff
@@ -209,8 +209,11 @@ impl ViewComponentParam {
             .max(0.000000001); // avoid division by 0
 
         f32::min(
-            max_size / sum_size,
-            max_alternative_size / max_alternative_size_for_child,
+            1.0,
```
Comment (wkozyra95) [inline compositor_render/src/scene/view_component/layout.rs:212]: unrelated fix (overflow=fit was fitting even if there was no overflow)

File: examples/image.rs [hunk 2]
```diff
@@ -103,148 +103,47 @@ fn start_example_client_code() -> Result<()> {
...
-                "background_color_rgba": "#0000FF00",
-                "resolution": { "width": 960, "height": 540 },
-                "children": [svg, label("SVG example", "svg_1_label")],
-            },
-        ],
+            new_image("example_png", "PNG example"),
+            new_image("example_jpeg", "JPEG example"),
+            new_image("example_svg", "SVG example"),
+            new_image("example_gif", "GIF example"),
+        ]
```
Comment (WojciechBarczynski) [inline examples/image.rs:146]: rm line `inputs` from `update_scene`

**Review by WojciechBarczynski — APPROVED** [review]
